README.md

```diff
@@ -64,6 +64,13 @@ package. Please refer to the README and instructions of those SDKs for more deta
   native crashes
 - [`@sentry/capacitor`](https://github.com/getsentry/sentry-capacitor): SDK for Capacitor Apps and Ionic with support
   for native crashes
+- [`@sentry/bun`](https://github.com/getsentry/sentry-javascript/tree/master/packages/bun): SDK for Bun
+- [`@sentry/deno`](https://github.com/getsentry/sentry-javascript/tree/master/packages/deno): SDK for Deno
+
+## Version Support Policy
+
+The current version of the SDK is 8.x. Version 7.x of the SDK will continue to receive critical bugfixes until end
+of 2024.
```

Comment:
```suggestion
The current version of the SDK is 8.x. Version 7.x of the SDK will continue to receive critical bugfixes until the end of 2024.
```

Reply:
this is auto-formatted via prettier, the newline is not visible :)

Reply:
Was about “the end of” mostly